admin-dev/themes/new-theme/js/app/utils/table-filters.js

```diff
@@ -21,24 +21,28 @@
  * @copyright 2007-2017 PrestaShop SA
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
- *}
+ */
 
-{extends file="helpers/list/list_header.tpl"}
+/**
+ * Allow to be able to filter table results according to columns desc/asc.
+ * This force a reload of page with more query parameters.
+ */
+global.TableFilters = {};
 
-{block name="override_header"}
+TableFilters.init = () => {
+    const url = new URL(window.location.href);
```

Comment:
Warning, URL is not available on IE+: how can I use a polyfill ? (I've already added the dependency in package.json file)